code/script.js

```diff
@@ -1,19 +1,108 @@
-// Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
+
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Welcome to my JavaScript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
 
+//UserName
+const userName = prompt("Please enter your name")
+
+alert(`Welcome ${userName}! Good to see you! Let's take a look at our different food options, shall we?`);
+
+
 // Step 2 - Food choice
-// Your code goes here
+const foodChoice = prompt(`What would you like today?
+1 - Pizza
+2 - Salad
+3 - Pasta
+Please enter the number of your choice.`);
+
+let selectedFood = "";
+
+if (foodChoice === "1") {
+  selectedFood = "Pizza";
+} else if (foodChoice === "2") {
+  selectedFood = "Salad";
+} else if (foodChoice === "3") {
+  selectedFood = "Pasta";
+} else {
+  alert("Invalid option or input! Please select a valid number :)");
+  process.exit(1);
```

Comment:
Good solution for exit, could also be "throw new Error()". Inside () you could put a costum an (error) message, which would show up in console.